apps/framework-docs/src/pages/building/data-models/model-data.mdx

```diff
@@ -2,54 +2,63 @@ import { Cards, Card, Tabs } from "nextra/components";
 
 ## Creating a Model
 
-To get started you simply need to open the `models.prisma` file in your project's `datamodels` directory and add a new Data Model definition.
-
-```prisma filename="datamodels/models.prisma" copy
-model UserActivity {
-    eventId     String @id
-    timestamp   DateTime
-    userId      String
-    activityType String
+To get started you simply need to open the `models.ts` file in your project's `datamodels` directory
+and add a new Data Model definition.
+
+```ts filename="datamodels/models.ts" copy
+type Key<T extends string | number> = T;
+
+export interface UserActivity {
+  eventId: Key<string>;
+  timestamp: string;
+  userId: string;
+  activity: string;
 }
 ```
 
 ## Data Model conventions
 
-As you can see, the data model is very simple. It's just a list of fields and their types. MooseJS's data modeling is built on top of Prisma's intuitive schema modeling language. You can read more about Prisma's schema modeling language [here](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema). We don't currently support all of Prisma's schema modeling features, but we're working on it; and we’ll also soon be adding support to define Data Models as Typescript interfaces.
+As you can see, the data model is very simple. It's just a list of fields and their types.
+MooseJS's data modeling is integrated with the data modeling capabilities of your language of choice.
+For now that's Typescript Interfaces but Python is in the works, leveraging Data classes.
+We don't currently support all of Typescript schema modeling features, but we're working on it.
```

Comment:
I would add an explanation about the Key functionality here

Reply:
The `Key` as in `type Key<T extends string | number> = T;` or key functionality of ts data model in general

Reply:
maybe both? i would explain why that specific line of code is needed / what it does, and then I would explain what marking a property as a Key does. and maybe any limitations on the convention (like, is it 1 and only 1 property must be marked as Key? can the Key only be a string or number type?)

Reply:
Cool - I believe I addressed the feedback :) Thank you